palantir-java-format/build.gradle

```diff
@@ -95,3 +103,11 @@ jar {
         attributes('Add-Exports': exports.stream().collect(Collectors.joining(' ')))
     }
 }
+
+tasks.named('jar', Jar){
+    // hack to get java21 Class ...
```

Comment:
The hack should be explained more. Why is a jar needed, why does simply copying .class files help. I assume, you had to modify other places, too?

Reply:
the palantir-java-formart21 project needs the palantir-java-format project.
but I need to include the whole class from palantir-java-formart21 to palantir-java-formart because the palantir-java-format jar is included in many projects on the Internet.
I think it will be easier to move all the code from palantir-java-format to the new palantir-java-format17 module and make palantir-java-format an aggregator module with the whole class without compile it.

gradle should be update to fully support java 21.